docs/installation.rst

```diff
@@ -157,35 +157,57 @@ properly.
-| ALLOW_PUBLIC_PROJECT_CREATION |  ``True``                       | If set to `True`, everyone can create a project without entering the admin password    |
-|                               |                                 | If set to `False`, the password needs to be entered (and as such, defined in the       |
+| ALLOW_PUBLIC_PROJECT_CREATION |  ``True``                       | If set to ``True``, everyone can create a project without entering the admin password  |
+|                               |                                 | If set to ``False``, the password needs to be entered (and as such, defined in the     |
 |                               |                                 | settings).                                                                             |
+|                               |                                 |                                                                                        |
+|                               |                                 | **Production value:** Usually, you will want to set it to ``False`` for a personal     |
+|                               |                                 |  instance.                                                                             |
 +-------------------------------+---------------------------------+----------------------------------------------------------------------------------------+
 | ACTIVATE_ADMIN_DASHBOARD      |  ``False``                      | If set to `True`, the dashboard will become accessible entering the admin password     |
 |                               |                                 | If set to `True`, a non empty ADMIN_PASSWORD needs to be set                           |
+|                               |                                 |                                                                                        |
+|                               |                                 | **Production value:** Up to you, according to your needs.                              |
 +-------------------------------+---------------------------------+----------------------------------------------------------------------------------------+
 | APPLICATION_ROOT              |  ``""``                         | If empty, ihatemoney will be served at domain root (e.g: *http://domain.tld*), if set  |
 |                               |                                 | to ``"foo"``, it will be served from a "folder" (e.g: *http://domain.tld/foo*)         |
+|                               |                                 |                                                                                        |
+|                               |                                 | **Production value:** Up to you, according to your needs.                              |
```

Comment:
I don't think this adds much value, we could just remove the "production value" for this setting.